[02062022 05:00:29] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[02062022 05:00:29] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[02062022 05:01:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:02:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:03:29] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 05:03:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:04:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:04:58] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:04:58] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:05:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:06:26] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 05:06:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:07:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:08:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:09:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:09:46] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 05:10:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:11:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:12:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:13:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:14:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:15:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:16:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:17:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:18:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:19:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:20:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:21:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:21:57] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 05:21:57] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 05:22:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:23:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:24:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:25:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:26:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:27:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:28:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:29:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:30:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:30:54] tftp /opt/dionaea/lib/dionaea/python/dionaea/tftp.py:719-warning: Requested file /opt/dionaea/var/dionaea/roots/tftp/ay9mfwq7xxmd4w6cz does not exist.
[02062022 05:30:54] tftp /opt/dionaea/lib/dionaea/python/dionaea/tftp.py:723-warning: Requested file not found: /opt/dionaea/var/dionaea/roots/tftp/ay9mfwq7xxmd4w6cz
[02062022 05:31:31] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 05:31:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:32:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:32:53] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:32:53] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:33:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:34:13] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 05:34:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:35:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:36:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:37:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:38:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:39:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:40:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:41:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:42:31] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:42:31] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:42:32] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:42:43] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:42:43] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:43:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:43:56] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:43:56] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:44:03] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:44:03] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:44:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:44:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:44:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:44:45] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 05:45:18] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:45:18] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:45:24] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:45:24] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:45:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:46:07] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:46:07] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:46:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:47:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:47:41] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:47:41] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:48:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:49:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:50:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:51:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:52:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:53:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:54:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:55:09] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 05:55:09] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 05:55:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:56:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:57:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:58:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 05:59:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:00:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:01:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:02:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:03:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:04:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:05:11] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:05:11] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:05:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:06:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:07:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:08:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:09:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:10:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:11:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:12:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:13:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:14:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:15:11] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:15:11] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:15:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:16:29] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 06:16:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:17:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:17:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:17:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:17:46] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:17:46] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:18:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:19:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:20:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:21:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:22:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:23:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:24:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:25:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:26:01] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:26:01] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:26:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:27:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:27:52] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 06:28:22] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:28:22] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:28:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:29:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:30:13] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 06:30:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:30:44] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:30:44] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 06:31:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:32:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:33:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:34:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:35:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:36:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:37:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:37:37] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:37] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 06:37:38] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 06:38:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:39:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:40:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:41:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:42:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:43:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:44:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:45:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:46:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:47:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:48:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:49:41] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 06:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:57:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:58:08] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 06:58:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 06:59:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:00:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:01:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:01:51] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 07:01:51] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 07:01:57] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:127-warning: p.haslayer(Raw): b'PATH_REPLACE__?????\x00'
[02062022 07:01:57] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:157-warning: p.haslayer(Raw): b'PATH_REPLACE__?????\x00'
[02062022 07:02:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:03:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:04:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:05:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:06:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:07:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:08:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:09:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:10:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:11:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:12:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:13:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:14:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:15:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:16:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:17:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:18:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:19:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:20:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:21:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:22:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:23:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:24:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:24:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 07:24:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 07:25:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:26:17] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 07:26:17] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 07:26:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:27:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:27:51] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 07:27:51] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 07:28:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:29:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:30:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:31:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:32:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:33:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:34:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:35:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:36:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:37:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:38:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:39:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:40:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:41:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:42:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:43:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:44:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:45:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:46:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:47:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:48:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:49:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:52:52] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 07:52:52] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 07:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:54:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:55:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:56:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:57:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:58:01] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 07:58:01] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening b''
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:261-warning: SQL ERROR no such function: VERSION
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:262-warning: SQL ERROR in b'SELECT VERSION()'
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:261-warning: SQL ERROR cannot rollback - no transaction is active
[02062022 07:58:02] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:262-warning: SQL ERROR in b'rollback'
[02062022 07:58:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 07:59:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:00:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:01:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:02:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:03:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:04:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:05:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:05:55] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 08:05:55] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 08:06:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:07:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:08:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:09:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:10:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:11:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:12:00] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 08:12:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:13:19] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 08:13:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:14:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:14:38] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 08:15:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:16:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:17:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:18:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:19:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:20:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:21:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:22:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:23:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:24:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:25:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:26:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:27:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:28:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:29:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:30:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:31:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:32:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:33:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:34:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:35:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:36:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:37:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:38:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:39:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:40:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:41:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:42:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:43:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:44:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:45:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:46:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:47:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:48:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:57:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:58:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 08:59:25] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 08:59:25] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 08:59:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:00:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:00:51] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 09:00:51] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 09:01:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:02:22] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 09:02:22] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 09:02:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:03:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:04:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:05:09] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 09:05:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:06:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:06:42] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 09:07:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:08:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:09:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:10:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:11:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:12:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:13:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:14:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:15:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:16:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:17:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:18:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:19:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:20:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:21:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:22:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:23:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:24:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:25:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:26:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:27:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:28:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:29:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:30:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:31:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:32:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:33:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:34:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:35:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:36:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:37:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:38:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:39:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:40:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:41:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:42:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:43:13] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 09:43:13] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 09:43:31] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:44:32] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 09:44:32] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 09:44:32] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:45:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:45:51] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 09:46:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:47:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:48:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:57:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:58:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 09:59:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:00:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:01:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:02:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:03:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:04:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:05:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:06:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:07:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:08:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:09:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:10:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:11:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:12:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:13:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:14:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:15:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:16:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:17:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:18:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:19:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:20:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:21:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:22:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:23:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:24:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:25:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:26:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:27:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:28:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:29:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:30:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:31:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:31:58] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 10:31:58] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 10:32:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:33:24] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 10:33:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:34:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:34:46] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 10:34:46] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 10:35:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:36:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:37:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:38:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:39:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:40:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:41:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:42:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:43:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:44:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:45:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:46:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:47:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:48:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:54:47] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 10:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:55:53] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 10:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:56:37] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 10:56:37] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 10:57:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:58:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 10:58:44] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:127-warning: p.haslayer(Raw): b'PATH_REPLACE__?????\x00'
[02062022 10:58:44] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:157-warning: p.haslayer(Raw): b'PATH_REPLACE__?????\x00'
[02062022 10:59:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:00:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:01:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:02:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:03:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:04:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:05:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:06:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:07:32] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:08:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:09:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:10:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:11:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:12:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:13:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:14:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:15:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:16:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:17:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:18:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:19:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:20:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:21:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:22:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:23:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:24:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:25:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:26:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:27:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:28:28] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 11:28:28] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 11:28:28] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:28:28] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[02062022 11:28:28] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[02062022 11:28:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:28:33] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:29:06] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:29:09] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:29:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:30:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:31:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:32:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:33:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:34:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:35:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:36:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:37:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:38:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:39:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:40:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:41:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:42:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:43:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:43:40] tftp /opt/dionaea/lib/dionaea/python/dionaea/tftp.py:719-warning: Requested file /opt/dionaea/var/dionaea/roots/tftp/a.pdf does not exist.
[02062022 11:43:40] tftp /opt/dionaea/lib/dionaea/python/dionaea/tftp.py:723-warning: Requested file not found: /opt/dionaea/var/dionaea/roots/tftp/a.pdf
[02062022 11:44:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:45:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:46:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:47:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:48:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:53:28] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:53:28] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 11:53:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:54:21] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:54:21] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 11:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:54:50] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:54:50] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 11:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:55:50] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:55:50] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:55:56] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:56:11] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:56:11] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 11:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:56:43] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:56:43] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 11:57:07] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:57:07] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[02062022 11:57:19] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[02062022 11:57:19] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[02062022 11:57:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[02062022 11:57:44] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[02062022 11:57:44] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[02062022 11:58:33] sip /opt/dionaea/l